scripts/policy/frameworks/cluster/nodes-experimental/manager.zeek

```diff
@@ -0,0 +1,55 @@
+##! This script provides experimental features in the cluster manager.
```

Comment:
Could clarify here that it is only loaded on the manager node, so one doesn't look for `@if ( Cluster::local_node_type() == Cluster::MANAGER )`.